README.md

```diff
@@ -1,3 +1,41 @@
 # Tobira: an Opencast Video Portal
 
-The project is in its *very* early stages, there is nothing to see here yet.
+__The project is in its *very* early stages, there is nothing to see here yet.__
+
+## Structure
+
+Tobira consists of two parts:
+
+- `backend` contains the video portal backend written in Rust which takes care of data persistens and communication to Opencast.
```

Comment:
```suggestion
- `backend` contains the video portal backend written in Rust which takes care of data persistence and communication to Opencast.
```

Also, "implementing the GraphQL API" is probably another important part of the backend.